cmd/cadvisor.go

```diff
@@ -55,6 +57,11 @@ var httpAuthRealm = flag.String("http_auth_realm", "localhost", "HTTP auth realm
 var httpDigestFile = flag.String("http_digest_file", "", "HTTP digest file for the web UI")
 var httpDigestRealm = flag.String("http_digest_realm", "localhost", "HTTP digest file for the web UI")
 
+var tlsCertPath = flag.String("tls_cert_path", "", "TLS certificate file path")
+var tlsKeyPath = flag.String("tls_key_path", "", "TLS key file path")
+var tlsCAPath = flag.String("tls_ca_path", "", "TLS certificate authority file path")
+var tlsClientAuth = flag.String("tls_client_auth", "require", "TLS authentication mode, must be one of request, optional, requireany, require or none")
```

Comment:
Am I right that:
* `tlsCertPath` and `tlsKeyPath` are used for HTTPS purposed, and
* `tlsCAPath` and `tlsClientAuth` are used for authentication purposes?

If so it would be great to clearly state it on the flag usage description.

Reply:
Done. Lmk what you think of the wording. I don't words good